.github/workflows/performance-tests.yml

```diff
@@ -52,11 +52,29 @@ jobs:
         run: |
           cd ../driver
           sudo make web sql="${{ secrets.DB_CONN_STRING }}"
+      - name: Check out delphi-admin
+        uses: actions/checkout@v3
+        with:
+          repository: cmu-delphi/delphi-admin
+          token: ${{ secrets.GH_PERSONAL_TOKEN }}
+          path: delphi-admin
       - name: Build & run Locust
+        continue-on-error: true # sometimes ~2-5 queries fail, we shouldn't end the run if that's the case
```

Comment:
what are the failures?  they only happen transiently?

Reply:
There are two queries from the "v4 - small" set that consistently fail.